assignments/array-methods.js

```diff
@@ -56,21 +56,43 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(function (object){
+    let name = object.first_name + " " + object.last_name;
+    fullName.push(name)
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+runners.map(function (object,index,arr) {
+allCaps.push(arr[index].first_name.toUpperCase())   
+})
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
+
+const sizeFltr = function(object, index, arr) {
+    if (arr[index].shirt_size === "L") {
```

Comment:
you could shorten this function down to just `return arr[index].shirt_size === "L"` because what the if check is doing is looking for a truthy or falsey value. If truthy, it will run the code inside the if closure, if falsey it will skip. Because `arr[index].shirt_size === "L"` can only be true or false, you can skip the extra logic and get right to the answer.